src/web/calculators/shared/components/KvKaDisplay.tsx

```diff
@@ -0,0 +1,56 @@
+import SingleInputLine from "common/components/io/inputs/SingleInputLine";
+import MeasurementOutput from "common/components/io/outputs/MeasurementOutput";
+import Measurement from "common/models/Measurement";
+
+export interface KvKaDisplayProps {
+  kV: Measurement;
+  kA: Measurement;
+  angular?: boolean;
```

Comment:
May be worth turning this from a boolean into either a literal or an enum, eg `mode: "linear" | "angular"` or something (there's probably a better name than `mode`).

not a big deal at the end of the way either way.